### PR TITLE
SAK-46535 Resources : clarify secure flag text

### PR DESCRIPTION
**Pull request by bgarciaentornos**
Creating as a draft as texts have to be reviewed

### CODE REVIEW

**Comment by bjones86** [comment]
This makes sense, but I'm not sure about the wording. @fostersdesign ?

File: content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm [hunk 1]
```diff
@@ -548,6 +549,7 @@ button.btn.btn-default.multiselect-clear-filter{
 											/>
 											$tlang.getString("access.secured")
 										</label>
+										<a class="fa fa-info-circle" title="[$tlang.getString('access.secured.tooltip')] tabindex="0" data-placement="bottom" role="button" data-toggle="popover" data-trigger="focus" data-html="true" data-content="$tlang.getString('access.secured.tooltip')"></a>
```
Comment (ottenhoff) [inline content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm:552]: the quotes on the title attribute look off

**Review by mpellicer — APPROVED** [review]
Looks good to me @bgarciaentornos, thanks for the good work.

File: content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm (hunk 1 above)
Comment (mpellicer) [reply content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm]: This was addressed, thanks @ottenhoff

**Pull request merged**